AndelaOSP/art-backend
# #159023262 submitted_by field in incidence report response

## What does this PR do?
- adds a submitted_by field to incidence report

## Description of the task to be completed
- add the test for submitted_by on incidence report model/API
- add a submitted_by field to incidence report model

## How can this PR be manually tested
- make a `post` API call on `http://127.0.0.1:8000/api/v1/incidence-reports` using postman or tools like curl and check the response for `submitted_by field`
## Relevant Pivotal Tracker stories
[#159023262](https://www.pivotaltracker.com/story/show/159023262)
## Screenshot
<img width="1279" alt="screen shot 2018-07-17 at 9 32 40 am" src="https://user-images.githubusercontent.com/31279497/42806324-259fa24c-89a6-11e8-97c3-17a596ed6873.png">
